src/components/Strategies/StrategyList.tsx

```diff
@@ -85,8 +71,12 @@ const Container = styled.div`
     display: inline-block;
   }
 
-  .soon {
+  .comingSoonText {
```

Comment:
You still haven't set the start and end opacity. Does it actually animate like this? Whatever the case, I already explained that you need to explicitly set both start and end.